### ISSUE TITLE
Optionally surround header rows in thead tag to allow interoperability in between plug-ins

### ISSUE THREAD

**Issue by ascholtz** [cosmocode/csv#15]
Specifically the following two plugins: include + datatables

https://github.com/LotarProject/dokuwiki-plugin-datatables/issues/21

Thank you,

Andrej

**Comment by ascholtz** [comment]
I think I fixed it:
```
diff -r M:\Toolbox\Dev\DokuWiki\csv plugin\table.php M:\Toolbox\Dev\DokuWiki\csv plugin\table.php.patched
92,94d91
<             if($line == 0 && $opt['hdr_rows'] > 0 ) {
<                 $renderer->tablethead_open();
<             }
134,136d130
<             if($line == $opt['hdr_rows'] - 1 ) {
<                 $renderer->tablethead_close();
<             }
```

**Comment by splitbrain** [comment]
Can you send a pull request please?

**Comment by splitbrain** [comment]
fixed by #23
